applications/GeoMechanicsApplication/tests/test_settlement_workflow/ProjectParameters_stage4.json

```diff
@@ -19,7 +19,7 @@
             "materials_filename":       "MaterialParameters.json"
         },
         "time_stepping":              {
-            "time_step":                1.0,
+            "time_step":                0.2,
```

Comment:
For stage 4 we now have multiple steps (3.2, 3.6 and 4.0 so the adaptive scheme is scheming away!)